Update azure-expressroute.md (#4730)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/azure-expressroute.md

> Microsoft Defender for Endpoint is not supported in Azure Express Route.

> [!NOTE]

> > Microsoft does not recommend ExpressRoute for Microsoft 365, nor does it provide the best connectivity model for the service in almost all circumstances. As such, Microsoft authorization is required to use this connectivity model for Microsoft 365. Microsoft reviews every customer request and authorizes ExpressRoute for Microsoft 365 only in the rare scenarios where it is necessary. Please read the [ExpressRoute for Microsoft 365 guide](https://aka.ms/erguide) for more information and following comprehensive review of the document with your productivity, network and security teams, work with your Microsoft account team to submit an exception should you require. Unauthorized subscriptions trying to create route filters for Office 365 will receive an [error message](https://support.microsoft.com/kb/3181709).

Reviewer: ```suggestion
> We do not recommend ExpressRoute for Microsoft 365 because it does not provide the best connectivity model for the service in most circumstances. As such, Microsoft authorization is required to use this connectivity model for Microsoft 365. We review every customer request and authorize ExpressRoute for Microsoft 365 only in the rare scenarios where it is necessary. Please read the [ExpressRoute for Microsoft 365 guide](https://aka.ms/erguide) for more information and following a comprehensive review of the document with your productivity, network, and security teams, work with your Microsoft account team to submit an exception if needed. Unauthorized subscriptions trying to create route filters for Office 365 will receive an [error message](https://support.microsoft.com/kb/3181709).
```
